src/main/java/io/gravitee/elasticsearch/client/http/HttpClient.java

```diff
@@ -536,61 +534,6 @@ public Completable putPipeline(String pipelineName, String pipeline) {
             });
     }
 
-    /**
-     * Creates or updates an OpenSearch policy. This method is specifically designed for use with OpenSearch
-     * and is not compatible with Elasticsearch, as it uses OpenSearch-specific APIs for Index State Management (ISM).
-     **/
-    @Override
-    public Completable createOrUpdatePolicy(String policyName, String policy, String seqNo, String primaryTerm) {
-        logger.debug("Create or update policy: url [{}] name[{}] policy[{}]", URL_ISM_POLICIES + '/' + policyName, policyName, policy);
```

Comment:
I think you can also remove the URL_ISM_POLICIES

Reply:
Done thanks 🙏